src/Message/Formatter.php

```diff
@@ -141,7 +141,16 @@ private function defaultFormat(Message $message, array $commonContext): string
     private function getTime(Message $message): string
     {
         $timestamp = (string) $message->context('time', microtime(true));
-        $format = strpos($timestamp, '.') === false ? 'U' : 'U.u';
+        $format = 'U';
+
+        if (strpos($timestamp, '.') !== false) {
+            $format = 'U.u';
+        }
+
+        if (strpos($timestamp, ',') !== false) {
+            $format = 'U,u';
+        }
```

Comment:
```suggestion

        $format = 'U';
        if (strpos($timestamp, '.') !== false) {
            $format .= '.u';
        } elseif (strpos($timestamp, ',') !== false) {
            $format .= ',u';
        }
```

Reply:
Because adding a string if it is faster to assign directly, it is not necessary.

Reply:
Because calling a function if we know it's just a dot, or comma for the floating point decimal, we don't need to call the function, and fetch all the `setlocale()` settings.